File: boardservice/src/main/java/com/workingbit/board/config/BoardProperties.java
package com.workingbit.board.config;

import com.workingbit.coremodule.config.AppProperties;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

/**
 * Created by Aleksey Popryaduhin on 08:57 11/06/2017.
 */
@Component
public class BoardProperties extends AppProperties {

  @Value("${TEST}")
  private boolean test;

  @Value("${AWS_ENDPOINT}")
  private String endpoint;

  public boolean isTest() {
    return test;
  }

  public String getEndpoint() {
    return endpoint;
  }
}

File: boardmodule/src/main/kotlin/com/workingbit/boardmodule/config/BoardProperties.kt
package com.workingbit.boardmodule.config

import com.workingbit.coremodule.config.AppProperties

/**
 * Created by Aleksey Popryaduhin on 19:25 25/08/2017.
 */
class BoardProperties : AppProperties() {

    val test: Boolean = true

    val endpoint: String = "http://localhost:8083"
}
